Readme.md

```diff
@@ -1,29 +1,25 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128592677/23.2.1%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E689)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [BlazorEnumPropertyEditor](./CS/EFCore/EnumCheckBoxEF/EnumCheckBoxEF.Blazor.Server/Editors)
-* [WinEnumPropertyEditor.cs](./CS/EFCore/EnumCheckBoxEF/EnumCheckBoxEF.Win/Editors/EnumPropertyEditorEx.cs) 
-* [DemoObjects.cs](./CS/EFCore/EnumCheckBoxEF/EnumCheckBoxEF.Module/BusinessObjects/DemoObjects.cs)
-* [TestFlagsAttributeEnum.cs](./CS/EFCore/EnumCheckBoxEF/EnumCheckBoxEF.Module/BusinessObjects/TestFlagsAttributeEnum.cs)
-<!-- default file list end -->
-# How to represent an enumeration property via a drop-down box with check boxes
 
+# XAF - How to Display an Enumeration Property as a Drop-down Box with Check Boxes
 
-<p><strong>Scenario:</strong></p>
-<p>There is an enumeration type decorated with the FlagsAttribute, which means that an enumeration can be treated as a set of flags. There is also a property of this enumeration type inside the business class. This is helpful when several predefined enumeration values can be stored using this data property. This example shows how to create an editor to represent such a property via an editor with multiple check boxes:</p>
+This example demonstrates how to create a custom Property Editor that displays an enumeration property as a drop-down box with check boxes, if the enumeration type has `FlagsAttribute` that allows to treat this enumeration as a set of flags.
```

Comment:
It's a pretty long sentence and I broke it down into two. 

The word "FlagsAttribute" needs an article. First of all, you don't refer to the type in general, you refer to an individual attribute instance assigned to a type. So you need an indefinite article... like when you would say "add a FlagsAttribute attribute to the type". (You kind of add a new instance or create something new.) But yes, then you want to remove repetition of the word "attribute". That leaves you with "a FlagsAttribute"  that reads like "a Flags attribute". 

```suggestion
This example demonstrates how to create a custom Property Editor that displays an enumeration property as a check list. The enumeration type must be declared with a `FlagsAttribute` that allows to treat this enumeration as a set of flags.
```